include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -61,6 +61,13 @@ static const char* const kOrtSessionOptionsEnableQuantQDQCleanup = "session.enab
 // GeluApproximation has side effects which may change the inference results. It is disabled by default due to this.
 static const char* const kOrtSessionOptionsEnableGeluApproximation = "optimization.enable_gelu_approximation";
 
+#ifdef ENABLE_TRAINING
+// Specifies a list of op types for memory footprint reduction.
+// The value should be a ","-delimited list of pair of <op type, alleviation strategy>.
+// For example, "Gelu:1,Dropout:1,Tile:2". Alleviation strateg currently has valid values: 0 - disabled, 1 - recompute.
```

Comment:
nit: type strateg --> strategy

can you add comment for value 2

Reply:
Right now we only support Gelum Dropout and Tile ops right? Can you mention this in the comment as well?

Reply:
> 

Oh, I just removed 2, because we did not support other approach than recompute.

Reply:
> 

Just added.